src/utils/errorTracking.ts

```diff
@@ -0,0 +1,29 @@
+import * as SentryExpo from "sentry-expo";
+
+const SENTRY_LOG_PREFIX = "[dev-sentry]";
+
+// This dev mock only implements the 4 currently used Sentry functions
+let Sentry: Pick<
+  typeof SentryExpo,
+  "init" | "setRelease" | "addBreadcrumb" | "captureException"
+>;
+
+if (__DEV__) {
+  Sentry = {
+    init: (): void => console.log(`${SENTRY_LOG_PREFIX} initialised`),
+    setRelease: (release: string): void =>
+      console.log(`${SENTRY_LOG_PREFIX} setRelease: ${release}`),
+    addBreadcrumb: (breadcrumb: unknown): void =>
+      console.log(
+        `${SENTRY_LOG_PREFIX} addBreadcrumb: ${JSON.stringify(breadcrumb)}`
+      ),
+    captureException: (e: unknown): string => {
+      console.warn(`${SENTRY_LOG_PREFIX} captureException: ${e}`);
+      return "eventId";
+    }
+  };
+} else {
+  Sentry = SentryExpo;
```

Comment:
Hmm, are we just initialising it for dev first to try? 

Just curious why we're not configuring more if its not dev HAHA

Reply:
Hmm, not very sure what you mean. The goal of this abstraction is to make it easier to see which Sentry function is being called during development, and also to avoid issues where an error pops up saying that Sentry is not initialised (when it doesn't matter in dev).

If you're referring to the configuration, we initialise it in `App.tsx`, other than that no other configuration is needed.

Reply:
Ahh, okok, got it! :D Thanks for explaining haha